agent/bench-scripts/postprocess/trafficgen-postprocess-cdm

```diff
@@ -23,11 +23,12 @@ my $rundir = shift(@ARGV);
 my $base_bench_dir = shift(@ARGV);
 
 my @trial_metrics = (
-    { 'key' => 'result',
+    { 'key' => 'flubberbubbles',
```

Comment:
@k-rister is this a technical term?

Reply:
Why of course.  Actually, this particular metric was an exception/one-off so I tagged it with something weird so it would never match.

Reply:
Note, this will probably change if we come up with the different metric value documents.  The difference is this is the pass/fail value as opposed to a double.